portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,25 +39,34 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    Integer numComments = Integer.parseInt(request.getParameter("quantity"));
+
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     Query query = new Query("Comment").addSort("dateTime", SortDirection.DESCENDING);
     PreparedQuery results = datastore.prepare(query);
 
     List<Comment> comments = new ArrayList<>();
 
     for (Entity entity : results.asIterable()) { 
-        String userFirstName = (String) entity.getProperty("firstName");
-        String userLastName = (String) entity.getProperty("lastName");
-        String dateTime = (String) entity.getProperty("dateTime");
-        String userMessage = (String) entity.getProperty("message");
-
-        Comment userComment = new Comment(userFirstName, userLastName, dateTime, userMessage);
-        comments.add(userComment);
+      String userFirstName = (String) entity.getProperty("firstName");
+      String userLastName = (String) entity.getProperty("lastName");
+      String dateTime = (String) entity.getProperty("dateTime");
+      String userMessage = (String) entity.getProperty("message");
+      Comment userComment = new Comment(userFirstName, userLastName, dateTime, userMessage);
+      comments.add(userComment);
     }
     
     response.setContentType("application/json;");
     Gson gson = new Gson();
-    response.getWriter().println(gson.toJson(comments));
+
+    //Print number of comments depending on numComments value
```

Comment:
nitty nit: please leave a blank space after `//`

Reply:
Good spot, thank you!